### ISSUE TITLE
About results

### ISSUE THREAD

**Issue by gonhjj** [sfimediafutures/CLIPping-the-Deception#15]
Author, may I ask if the impact of using different devices or modifying random seeds on the results of your project is significant?

**Comment by sohailahmedkhan** [comment]
Just changing random seed or using a different device for training would not cause significant changes in results. However, minor deviations can still occur.

**Comment by gonhjj** [comment]
> 仅更改随机种子或使用不同的设备进行训练不会导致结果发生重大变化。但是，仍可能发生微小的偏差。

Is a 3% -4% increase or decrease normal?

**Comment by gonhjj** [comment]
> > 仅更改随机种子或使用不同的设备进行训练不会导致结果发生重大变化。但是，仍可能发生微小的偏差。
> 
> Is a 3% -4% increase or decrease normal?

I forgot to mention the AVG-Acc indicator

**Comment by sohailahmedkhan** [comment]
Are you training from scratch? 3-4% does not seem to be normal.

**Comment by gonhjj** [comment]
> Are you training from scratch? 3-4% does not seem to be normal.

Yes, but I haven't found what caused it yet.

**Comment by gonhjj** [comment]
@sohailahmedkhan Can you talk about the hyperparameter settings you researched? The results in the paper show how many rounds of training were conducted. Thanks.

**Comment by gonhjj** [comment]
Thank you very much, I have found the areas that need to be modified.
